src/turn_integrator.py

```diff
@@ -85,15 +109,92 @@ def send_media_message(msisdn, media_type, media_id, line_name, caption=""):
             "caption": caption
         }
 
-    message_data["recipient_type"] = "individual"
-    message_data["to"] =  msisdn
-    response = send_message(message_data, line_name)
+    response = send_message(line_name, message_data)
     print(response.text)
     return response
 
-### Save media to Turn for sending. See the supported file types on the Turn documentation here:
-### https://whatsapp.turn.io/docs/api/media#supported-file-types
-def save_media(type, file_binary, line_name):
+
+## Send an interactive message with a dropdown or buttons.
+## The sections argument is a dictionary with a few attrbutes:
+## 'header_text' - text, optional
+## 'header_image' - id of a saved image, optional
+## 'body_text' - text, required
+## 'footer_text' - text, optional
+## 'buttons' - array, required array of buttons with text and callback_id, used if the interactive_type
+## is a 'button'.
+## 'list_button' - text, required text of the list button for interactive_type = 'list'
+## 'list_title' - text, required title text for the list for interactive_type = 'list'
+## 'list_items' - array, required array of items with text and callback_id for interactive_type = 'list'
+## Further details about the API call here:
+## https://whatsapp.turn.io/docs/api/messages#interactive-messages
+
+def send_interactive_message(msisdn: str, line_name: str, interactive_type: str, sections: json) -> requests.Response:
+    message_data = {
+        "to": msisdn,
+        "type": "interactive",
+        "interactive": {
+            "type": interactive_type,
+            "body": {
+                "text": sections['body_text']
+            },
+            'action': {}
+        },
+    }
+
+    if sections['header_text']:
+        message_data['interactive']['header'] = {
+            "type": "text",
+            "text": sections['header_text']
+        }
+    elif sections['header_image']:
```

Comment:
Are these two exclusive? You can't have a `header_image` and `header_text`?

Reply:
Seems from the Turn examples, yes.